app/views/tradingName/TradingNameChangeSuccessView.scala.html

```diff
@@ -15,13 +15,8 @@
  *@
 
 @import models.viewModels.ChangeSuccessViewModel
-@import common.ContactPreference
-@import views.html.templates.AgentPaperUpdateClient
-@import views.html.templates.UpdateClient
 
 @this(mainTemplate: MainTemplate,
-      updateClient: UpdateClient,
-      agentPaperUpdateClient: AgentPaperUpdateClient,
       govukButton: GovukButton)
 
 @(model: ChangeSuccessViewModel, isRemoval: Boolean, isAddition: Boolean)(implicit messages: Messages, appConfig: config.AppConfig, user: User[_])
```

Comment:
Changes all look good - could you also remove "isAddition" parameter as it's no longer being used